tests/utils/apis/test_pole_emploi_api.py

```diff
@@ -19,14 +19,13 @@
 from itou.utils.mocks import pole_emploi as pole_emploi_api_mocks
 from tests.job_applications.factories import JobApplicationFactory
 from tests.users.factories import JobSeekerFactory
-from tests.utils.test import TestCase
 
 
-class PoleEmploiAPIClientTest(TestCase):
+class TestPoleEmploiAPIClient:
     CACHE_EXPIRY = 3600
 
-    def setUp(self) -> None:
-        super().setUp()
+    @pytest.fixture(autouse=True)
```

Comment:
Pas la peine d’en faire une fixture: https://docs.pytest.org/en/stable/how-to/xunit_setup.html

```suggestion
```